sandy/core/lpc.py

```diff
@@ -244,6 +245,12 @@ def reshape(self, eg, selected_mat=None, selected_mt=None, inplace=False):
             MT number for which the reshape will apply (all MT by default)
         inplace : `bool`, optional, default is `False`
             flag to activate inplace replacement
+        kind: `str` or `int`, optional
```

Comment:
`kind` should not be present...the interpolation must be linear, as mentioned in the docstring description! this is because for LPC the ENDF-6 format states that the interpolation is linear